types/src/on_chain_config/timed_features.rs

```diff
@@ -11,8 +11,10 @@ use strum_macros::{EnumCount as EnumCountMacro, EnumIter};
 #[derive(Debug, EnumCountMacro, EnumIter, Clone, Copy, Eq, PartialEq)]
 pub enum TimedFeatureFlag {
     DisableInvariantViolationCheckInSwapLoc,
-    LimitTypeTagSize,
-    ModuleComplexityCheck,
+    // Was always enabled.
+    _LimitTypeTagSize,
```

Comment:
Presumably you are keeping these for the tests, but it still seems confusing at first glance, especially with the `_` prefix.

Shall we rename these to something like `TestFeature1` and `TestFeature2`, making the intention clear?

Reply:
I kept them in case we want to come back, but otherwise we can probably remove this? And have test feature separately?

Reply:
I guess I will keep them for now, and then we can decide if e want to remove or rename these